<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(30, 30, 30)

}
QLabel{
color: rgb(200, 200, 200);
}

QDoubleSpinBox{
background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;

}
QComboBox {
background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
}
QPushButton {
background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 10px;
padding: 1px;
}
QPushButton:hover {
background-color:rgb(20, 115, 230);
	color: rgb(200, 200, 200);
border: 1px solid rgb(20, 115, 230);
}
QPushButton:pressed {
background-color:rgb(18, 100, 200);
color: rgb(200, 200, 200);
border: 1px solid rgb(18, 100, 200);
}
QPushButton:disabled {
background-color:rgb(40, 40, 40);
color: rgb(100, 100, 100);
border: 2px solid rgb(50, 50, 50);
border-radius: 10px;
}

QLineEdit {
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 10px;
padding: 1px;
}

QLineEdit:focus {
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(20, 115, 230);
border-radius: 10px;
padding: 1px;
}

QProgressBar {
background-color:rgb(30, 30, 30);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
color:rgb(200, 200, 200);
text-align: center;
}
QProgressBar::chunk {
background-color:rgb(20, 115, 230);
border: 2px solid rgb(20, 115, 230);
border-radius: 1px;
}

QGroupBox {
color: rgb(200, 200, 200);
background: rgb(30, 30, 30);
border: 1px solid rgba(80, 80, 80, 255);
border-radius: 15px;
} 

</string>
  </property>
  <widget class="QGroupBox" name="gbox_settings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>  Settings  </string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lbl_tracker_dimensions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Tracker - Dimensions</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_width">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> px</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>400.000000000000000</double>
    </property>
    <property name="maximum">
     <double>900.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_width">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_height">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_height">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> px</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>400.000000000000000</double>
    </property>
    <property name="maximum">
     <double>900.000000000000000</double>
    </property>
    <property name="value">
     <double>900.000000000000000</double>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_d_factor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>D factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lens_corrections">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Lens Corrections</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_d_factor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_real_world">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Real World</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_RW_pos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Points</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_abs_distance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> m</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_RW_abdistance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Absolute distance
between two points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pos">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(40, 40, 40);
color: rgb(200, 110, 40);
border: 2px solid rgb(200, 110, 40);
border-radius: 10px;
padding: 1px;
}
QPushButton:hover {
background-color:rgb(200, 110, 40);
color: rgb(200, 200, 200);
border: 1px solid rgb(200, 110, 40);
}
QPushButton:pressed {
background-color:rgb(180, 100, 35);
color: rgb(200, 200, 200);
border: 1px solid rgb(180, 100, 35);
}</string>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_search_height">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> px</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>900.000000000000000</double>
    </property>
    <property name="value">
     <double>200.000000000000000</double>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tracker_height">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tracker_searchArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Tracker - Search area</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tracker_width">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_search_width">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> px</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>900.000000000000000</double>
    </property>
    <property name="value">
     <double>200.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_real_world_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Speed Limits</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SL_car">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>car</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_speed_motorbike">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> km/h</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SL_motorbike">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>motorbike</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_speed_car">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> km/h</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SL_bus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>bus</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SL_truck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>truck</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_speed_truck">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> km/h</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_speed_bus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string> km/h</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_direction">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Vehicle direction</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBox_direction">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Toward the camera</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Away from the camera</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_accept">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:rgb(40, 40, 40);
color: rgb(200, 0, 10);
border: 2px solid rgb(200, 0, 10);
border-radius: 10px;
padding: 1px;
}
QPushButton:hover {
background-color:rgb(190, 0, 10);
color: rgb(200, 200, 200);
border: 1px solid rgb(190, 0, 10);
}
QPushButton:pressed {
background-color:rgb(170, 0, 10);
color: rgb(200, 200, 200);
border: 1px solid rgb(170, 0, 10);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
